### PR TITLE
[NEW] Package to render issue numbers into links to an issue tracker.

### PR DESCRIPTION
**Pull request by TobiasKappe**
This PR adds the package "rocketchat-issuelinks", which ensures that numbers preceded by a #-sign get converted into issue links. The exact link is configurable. 

This package is incompatible with the "Hex Color Preview" plugin, which also uses the #-sign as an anchor to recognize numbers. As such, this feature is turned off by default.

@RocketChat/core

### CODE REVIEW

File: packages/rocketchat-issuelinks/package.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Package.describe({
+	name: 'rocketchat:issuelinks',
+	version: '0.0.1',
+	summary: 'Message pre-processor that will add links to issue numbers.',
```
Comment (TAdeJong) [inline packages/rocketchat-issuelinks/package.js]: Suggestion: 'Message pre-processor that adds links to issue numbers.'

**Comment by TAdeJong** [comment]
Alternative use: easily link relevant xkcd's!

**Comment by TobiasKappe** [comment]
Any feedback?

**Comment by wampiedriessen** [comment]
This is a feature I would like to see, and a vital package for e.g. a development team! :)

**Comment by jasonfah** [comment]
Would it be unreasonable to ask for the key (# in this case) to be configurable?

For example for this to work with our case management system it would detect...

Case 123

Where it matches on case[space][0-9]

**Comment by ulope** [comment]
This is a great start but unfortunately completely impractical for larger teams. In our company teams are working on 5 different projects. The minimum for a usable solution would be channel specific templates.

**Pull request merged**